serverless/getting_started/installing-openshift-serverless.adoc

```diff
@@ -6,47 +6,53 @@ include::modules/common-attributes.adoc[]
-
-In the size requirement provided, an application can scale up to 10 replicas. Lowering the actual CPU request of applications can increase the number of possible replicas.
+[id="installing-openshift-serverless-cluster-sizing-reqs"]
+== {product-title} cluster sizing prerequisites
 
-You can use the MachineSet API to manually scale your cluster up to the desired size. The minimum requirements usually mean that you must scale up one of the default MachineSets by two additional machines.
-
-For more information on using the MachineSet API, see the documentation on xref:../../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-aws[Creating MachineSets].
-
-For more information on scaling a MachineSet manually, see the documentation on xref:../../machine_management/manually-scaling-machineset.adoc#manually-scaling-machineset[manually scaling MachineSets].
+The minimum requirement to use {ServerlessProductName} is an {product-title} cluster with 10 CPUs and 40GB memory.
 
+The total size requirements to run {ServerlessProductName} depend on the CPU request limits configured for deployed applications.
+In the size requirement provided, an application can scale up to 10 replicas.
+Lowering the actual CPU request of applications can increase the number of possible replicas.
+// Question: how does a user/admin lower the CPU request of apps?
 [NOTE]
 ====
 The requirements provided relate only to the pool of worker machines of the {product-title} cluster. Master nodes are not used for general scheduling and are omitted from the requirements.
 ====
+// TODO: Update master and worker here  - to what?
 
-[NOTE]
-====
-The following limitations apply to all {ServerlessProductName} deployments:
+[id="installing-openshift-serverless-cluster-scaling"]
+=== Using MachineSets to scale {product-title} clusters
 
-* Maximum number of Knative services: 1000
-* Maximum number of Knative revisions: 1000
-====
+You can use the MachineSet API to scale your cluster up to the required size.
+It is recommended to scale up one of the default MachineSets by two additional machines.
+
+For more information on using the MachineSet API, see the documentation on xref:../../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-aws[Creating MachineSets].
+
+For more information on scaling a MachineSet manually, see the documentation on xref:../../machine_management/manually-scaling-machineset.adoc#manually-scaling-machineset[manually scaling MachineSets].
 
-=== Additional requirements for advanced use-cases
+[id="installing-openshift-serverless-cluster-sizing-reqs-advanced"]
+=== Additional cluster sizing requirements for advanced use cases
 
 For more advanced use-cases such as logging or metering on {product-title}, you must deploy more resources. Recommended requirements for such use-cases are 24 CPUs and 96GB of memory.
```

Comment:
s/96GB/96 GB/
For consistency with this doc and ISG, could you change this also?